hxmath/math/Vector2.hx

```diff
@@ -839,5 +839,20 @@ abstract Vector2(Vector2Type) from Vector2Type to Vector2Type
         return self.clone()
             .rotateRight();
     }
+
+    private inline function set_length(v:Float):Float
+    {
```

Comment:
I don't think this should be a property setter. It is an operation that can potentially cause an error (e.g. if set the length of the zero vector).

Reply:
Good point about the potential error. Would it be acceptable if I added in some logic to prevent the error? In my opinion the convenience of the setter is worth the small bit of overhead of checking if the vector is zeroed out before normalizing it.

Reply:
Removed `set_length()` method.